intel-lustre-clients-on-centos/azuredeploy.json

```diff
@@ -0,0 +1,403 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "artifactsBaseUrl": {
+            "type": "string",
+            "defaultValue":"https://raw.githubusercontent.com/arsenvlad/azure-quickstart-templates/master/intel-lustre-clients-on-centos",
+            "metadata": {
+                "description": "Base URL of the solution template gallery package",
+                "artifactsBaseUrl": ""
+            }
+        },
+        "location": {
+            "type": "string",
+            "allowedValues": [
+                "westus",
```

Comment:
The create UI def doesn't work using this file though. Actually the allowed values can probably be removed now since compute RP is available in all locations except newly launched DCs in India.